### ISSUE TITLE
cr-html fails for unfinished sessions

### ISSUE THREAD

**Issue by tomato42** [sixty-north/cosmic-ray#492]
when the session has not completed (`cosmic-ray exec` didn't finish), running `cr-html` fails:
```
Traceback (most recent call last):
  File "./cosmic-ray-venv/bin/cr-html", line 11, in <module>
    load_entry_point('cosmic-ray==5.6.2', 'console_scripts', 'cr-html')()
  File "/home/tomato/workspace/python-ecdsa/cosmic-ray-venv/lib/python3.7/site-packages/cosmic_ray-5.6.2-py3.7.egg/cosmic_ray/tools/html.py", line 26, in report_html
    arguments['--skip-success'])
  File "/home/tomato/workspace/python-ecdsa/cosmic-ray-venv/lib/python3.7/site-packages/cosmic_ray-5.6.2-py3.7.egg/cosmic_ray/tools/html.py", line 168, in _generate_html_report
    if not result.is_killed:
AttributeError: 'NoneType' object has no attribute 'is_killed'
```

**Comment by abingham** [comment]
We're just missing a guard around code like that; incomplete work-items won't have a result. Honestly, looking at that file, it could use some cleanup, maybe some refactoring to make it more clear where results are needed and where just the work-item is needed.

**Comment by disouzam** [comment]
Hello, @abingham :

I started using cosmic-ray today. First of all, thanks for your work and the work of other contributors.

After a quick start looking at the documentation, I noticed that the argument `--skip-success` is missing from the documentation. 

<img width="1403" height="733" alt="Image" src="https://github.com/user-attachments/assets/9b166403-c42e-49af-8090-f7de5e6cc87e" />

I was able to identify it in the code and even in the trace shown here this argument is presented.

In my case, this is a completely valid command:

```bash
cr-html tutorial.sqlite --skip-success
```
